### ISSUE TITLE
Build error on Zeek 5.0.10

### ISSUE THREAD

**Issue by Sithari** [cisagov/icsnpp-synchrophasor#10]
# 🐛 Summary #

Build failed.
```
root@ubuntu-linux-22-04-02-desktop:/usr/local/zeek/bin# ./zeek -v
./zeek version 5.0.10
```

## To reproduce ##

```
zkg refresh
zkg install icsnpp-synchrophasor
```

## Expected behavior ##

Excepted it to get installed, but it did not. 

## Any helpful log output or screenshots ##

Paste the results here:

```
root@ubuntu-linux-22-04-02-desktop:/usr/local/zeek/bin# ./zkg refresh
Refresh package source: zeek
	No membership changes
Refresh installed packages
	No new outdated packages
root@ubuntu-linux-22-04-02-desktop:/usr/local/zeek/bin# ./zkg install icsnpp-synchrophasor
The following packages will be INSTALLED:
  zeek/cisagov/icsnpp-synchrophasor (main)

Proceed? [Y/n] y
Running unit tests for "zeek/cisagov/icsnpp-synchrophasor"
error: failed to run tests for zeek/cisagov/icsnpp-synchrophasor: package build_command failed, see log in /usr/local/zeek/var/lib/zkg/logs/icsnpp-synchrophasor-build.log
Proceed to install anyway? [N/y] y
Installing "zeek/cisagov/icsnpp-synchrophasor"..
Failed installing "zeek/cisagov/icsnpp-synchrophasor": package build_command failed, see log in /usr/local/zeek/var/lib/zkg/logs/icsnpp-synchrophasor-build.log
error: incomplete installation, the follow packages failed to be installed:
  zeek/cisagov/icsnpp-synchrophasor (main)
root@ubuntu-linux-22-04-02-desktop:/usr/local/zeek/bin# cat /usr/local/zeek/var/lib/zkg/logs/icsnpp-synchrophasor-build.log
=== STDERR ===

====================|  Spicy-side Zeek Installation Summary  |====================

Found Zeek:            yes

Version:               5.0.10 (50010)
Prefix:                /usr/local/zeek
Build type:            relwithdebinfo

========================================================================


====================|  Spicy Installation Summary  |====================

Found Spicy:           yes

Version:               1.5.4 (10504)
Prefix:                /usr/local/zeek
Build type:            release
Have toolchain:        yes
Spicy compiler:        /usr/local/zeek/bin/spicyc

========================================================================

[error] /usr/local/zeek/var/lib/zkg/clones/package/icsnpp-synchrophasor/analyzer/synchrophasor.evt:6: expected 'import', '{file,protocol} analyzer', or 'on'
[error] error loading EVT file "/usr/local/zeek/var/lib/zkg/clones/package/icsnpp-synchrophasor/analyzer/synchrophasor.evt"
gmake[2]: *** [analyzer/CMakeFiles/SYNCHROPHASOR.dir/build.make:78: spicy-modules/synchrophasor.hlto] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:98: analyzer/CMakeFiles/SYNCHROPHASOR.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
=== STDOUT ===
-- The C compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- spicyz: /usr/local/zeek/bin/spicyz
-- Zeek plugin version: 1.3.26
-- Zeek plugin CMake path: /home/parallels/Downloads/zeek-5.0.10/build/src/builtin-plugins/spicy-plugin/cmake
-- Found zeek-config: /usr/local/zeek/bin/zeek-config
-- Found Zeek: yes  
-- Found spicy-config: /usr/local/zeek/bin/spicy-config
-- Found Spicy: yes  
-- spicyz: /usr/local/zeek/bin/spicyz
-- Found SpicyPlugin: /usr/local/zeek/bin/spicyz  
-- Configuring done
-- Generating done
-- Build files have been written to: /usr/local/zeek/var/lib/zkg/clones/package/icsnpp-synchrophasor/build
[ 50%] Compiling SYNCHROPHASOR analyzer

```

**Comment by mmguero** [comment]
Thanks, I will update the zkg requirement for the correct spicy version.

**Comment by mmguero** [comment]
FWIW, this plugin uses Spicy `export` syntax not supported in older versions of Spicy, so it will not support v5.

**Comment by mmguero** [comment]
Fixed with #11.

Zeek v6.0.0:

```
.root e82a63750657 zeek-logs> zeek --version
zeek version 6.0.0
.root e82a63750657 zeek-logs> zkg install /icsnpp-synchrophasor 
The following packages will be INSTALLED:
  /icsnpp-synchrophasor (main)

Proceed? [Y/n] y
Running unit tests for "/icsnpp-synchrophasor"
Installing "/icsnpp-synchrophasor"...........................
Installed "/icsnpp-synchrophasor" (main)
Loaded "/icsnpp-synchrophasor"
```

Zeek v6.1.0

```
.root ab1386c6c45e zeek-logs> zeek --version
zeek version 6.1.0
.root ab1386c6c45e zeek-logs> zkg install /icsnpp-synchrophasor
The following packages will be INSTALLED:
  /icsnpp-synchrophasor (main)

Proceed? [Y/n] y
Running unit tests for "/icsnpp-synchrophasor"
Installing "/icsnpp-synchrophasor"............................
Installed "/icsnpp-synchrophasor" (main)
Loaded "/icsnpp-synchrophasor"
```

Zeek v5.x

```
.root 37b710e9df6c zeek-logs> zkg install /icsnpp-synchrophasor 
error: failed to resolve dependencies: unsatisfiable dependency: "zeek" (5.0.10) is installed, but "/icsnpp-synchrophasor" requires >=6.0.0
```
